mappings/net/minecraft/world/gen/feature/SpruceTreeFeature.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_349 net/minecraft/world/gen/feature/SpruceTreeFeature
```

Comment:
Should actually be `PineTreeFeature`, according to wiki - https://minecraft.wiki/w/Spruce